cotainr/container.py

```diff
@@ -147,14 +147,27 @@ def add_metadata(self):
             f.seek(0)
             json.dump(metadata, f)
 
+    def _create_file(self, *, env_file):
+        """
+        Create the Path/file `env_file` in the Singularity container.
+
+        Parameters
+        ----------
+        env_file : :class:`pathlib.PosixPath`
+            For example, Path("sandbox_dir/.singularity.d/env/92-cotainr-env.sh")
+        """
+        self._assert_within_sandbox_context()
+
+        # ensure that the file is created *within* the container to get correct permissions, etc.
+        self.run_command_in_container(cmd=f"touch {env_file}")
+
+        if not env_file.exists():
+            raise FileNotFoundError(f"Creating file {env_file} failed.")
```

Comment:
[We don't seem to have a unit test covering this exception.](https://github.com/DeiC-HPC/cotainr/actions/runs/12010161900/job/33476424258#step:10:352)